src/parameterizations/lateral/MOM_lateral_mixing_coeffs.F90

```diff
@@ -1320,8 +1476,47 @@ subroutine VarMix_init(Time, G, GV, US, param_file, diag, CS)
                  units="m", default=-1.0, scale=GV%m_to_H)
     allocate(CS%ebt_struct(isd:ied,jsd:jed,GV%ke), source=0.0)
   endif
-  allocate(CS%BS_struct(isd:ied,jsd:jed,GV%ke), source=0.0)
-  CS%BS_struct(:,:,:) = 1.0
+
+
+  if (CS%BS_EBT_power>0. .and. CS%BS_use_sqg_struct) then
+    call MOM_error(FATAL, &
+      "calc_resoln_function: BS_EBT_POWER>0. &
```

Comment:
The indentation for a continuation line should be at least 4 spaces, according to the MOM6 style guide at https://github.com/NOAA-GFDL/MOM6/wiki/Code-style-guide#white-space.

Reply:
ISO Fortran also requires line continuations (`&`) on both the end and beginning of split strings, e.g.
```
print *, "abcd &
    & efgh"
```